### ISSUE TITLE
Display Alt Text when Media Hidden

### ISSUE THREAD

**Issue by mgifford** [mastodon/mastodon#20507]
### Steps to reproduce the problem

1. An image with alt text is no different than an image without alt text when the Media Hidden is hightlighted
2. When there is alt text, I should be able to read what it is before clicking to see the image
3. This should be better for low bandwidth users but also for encouraging people to use alt text.
...

### Expected behaviour

Display the alt text of images when the Media is Hidden

### Actual behaviour

No alt text was displayed

### Detailed description

Specific post is here https://mastodon.social/@mgifford/109331778770641062

![Screenshot with blurred images](https://user-images.githubusercontent.com/116832/201484451-c859f73d-a297-41e9-96c1-0c3f17fe2150.png)

### Specifications

Chrome

**Comment by mgifford** [comment]
Should add that this is when you have https://mastodon.social/settings/preferences/appearance

Media display set to "Always hide media":

![Screenshot 2022-11-12 12 20 21 PM](https://user-images.githubusercontent.com/116832/201486546-d119326e-f5bb-4995-8cd7-b2a17754a37e.png)

It makes this a bit less addictive (which is good).

I did wonder if the image is downloaded and then just blurred in the browser or if it would need to be retrieved from the server. I am looking for ways to work in low bandwidth environments (like at the lake without broadband).

**Comment by EdHamilton9** [comment]
I would like to have this feature also. If alt-text was visible while the image was hidden, I could decide if I wanted to view the image before clicking on it. 
My setting is the same as mgifford's, - "Always hide media"
